src/components/Drawer.js

```diff
@@ -131,31 +126,31 @@ const Drawer = React.createClass({
           type='base'
         />
       </nav>
-    ) : <div style={styles.nav} />;
+    ) : <div className={css(styles.nav)} />;
   },
 
   render () {
     const styles = this.styles();
```

Comment:
I'd like to see if creating the stylesheet on `componentWillMount` and then mixing inline styles when we need dynamic values would reduce / eliminate the need for the delay animation property.